backend/graph-proxy/src/graphql/workflows.rs

```diff
@@ -69,6 +69,13 @@ impl Workflow {
 
 #[Object]
 impl Workflow {
+    /// The unique ID derived from the visit and name
+    async fn id(&self) -> ID {
+        let visit_display = self.metadata.visit.to_string();
+        let unique_id = format!("{}:{}", visit_display, self.metadata.name);
+        ID::from(unique_id)
```

Comment:
This is sufficiently unique once other object types are queryable via the Node interface? That includes our own types, but potentially other services types in the federated graph.

Maybe to avoid clashes we should prefix with something like "<service_name>:<type_name>" to guarantee uniqueness?